### ISSUE TITLE
Cannot read property 'parent' of undefined

### ISSUE THREAD

**Issue by pradt** [jvandemo/angular-growl-notifications#29]
Hi I'm receiving the following error : Cannot Read property of 'parent' of undefined. 

My HTML : 

```
 <div class="notificationHolder">
        <div ng-repeat="(iid, notification) in notifications track by notification.iid" class="animNotifcation">



                <growl-notification ttl="5000">
                    <span ng-bind-html="notification.message"></span>
                </growl-notification>
            </div>
        </div>
    </div>
```

My Test event : 

```
$scope.$on("notification", function (event, args) {
            index++;
            args.notification.iid = index;
            args.notification.message += " " + index;
            $scope.notifications.push(args.notification);
            $timeout(function () {
                $scope.notifications.shift();
                //$scope.notifications.splice(index, 1);
            }, 5000);
        });
```

When I've de-bugged it in google chrome - I located the following line in the link function : 
$animate.move(iElem, growlNotifications.element);

and growlNotifications.element is undefined. 

I'm using Angular 1.5 with jquery 2.2.4 if that makes a difference.

**Comment by jvandemo** [comment]
@pradt &mdash; Do you have `<growl-notifications></growl-notifications>` (plural) defined in your markup as the target for your notifications to appear in? For example:

```
<!-- this tells the library where to render the notifications in the DOM -->
<growl-notifications></growl-notifications>

<div class="notificationHolder">
      <div ng-repeat="(iid, notification) in notifications track by notification.iid" class="animNotifcation">
            <growl-notification ttl="5000">
                <span ng-bind-html="notification.message"></span>
            </growl-notification>
        </div>
    </div>
</div>
```

Thanks!

**Comment by charly22** [comment]
I've stumbled upon this issue too and I've confirmed that when you don't have `<growl-notifications></growl-notifications>` and you tries to link a `<growl-notification>` to your template you get this error.

I based my code in this example http://jvandemo.github.io/angular-growl-notifications/examples/dynamically-add-notifications. Perhaps adding `<growl-notifications></growl-notifications>` to the example codebase or adding this to a FAQ would be helpful. Thanks
